stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/UserDto.kt

```diff
@@ -0,0 +1,11 @@
+package io.getstream.chat.android.client.api2.model.dto
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+internal data class UserDto(
+    var id: String,
+    var role: String,
+    var invisible: Boolean,
+    var banned: Boolean,
+)
```

Comment:
These model classes have a minimal set of data for now, for basic tests. Other properties will reference more models that have to be implemented (e.g. Attachments), plus the "custom object" behaviour and parsing is not implemented yet either.